src/index.js

```diff
@@ -7,6 +7,11 @@ const getBaseURI = () =>
     'IScript_'
   );
 
+const getUrlOrRedirect = url =>
+  dataURI && url.includes('IScript_')
+    ? url.replace(/.*IScript_/, dataURI)
```

Comment:
This replaces the correct stuff? It's going to leave part of the web library's name in the URL, e.g. `https://example.com/IScript_Foo?bar=true` would become `http://localhost/Foo?bar=true`.

That's the expected behavior?